images/fcos-bastion-image/root/etc/bastion.conf

```diff
@@ -0,0 +1,3 @@
+# systemd squid proxy conf
```

Comment:
Is it possible to move to a folder under /usr/?

In general, files under /usr should be preferred in this Containerfiles. When a per-deployment override should be considered, we can implement the logic such that some folder or file within /etc overrides the (default) content in /usr (not needed here IMO).

Also, can we rename from bastion.conf to something for squid?